python/common/org/python/types/Dict.java

```diff
@@ -357,10 +357,12 @@ public void __delitem__(org.python.Object item) {
 
     @org.python.Method(
             __doc__ = "",
-            args = {"iterable"},
-            default_args = {"value"}
+            default_args = {"iterable", "value"}
```

Comment:
@deep110 This looks tricky. This format of TypeError message seems to come from [cpython's getargs.c unpack_stack()](https://github.com/python/cpython/blob/6f0eb93183519024cb360162bdd81b9faec97ba6/Python/getargs.c#L2365), and it looks like it can be triggered elsewhere too, not just from Dict objects.

I think what we can do is to move the specific test that fails only due to the TypeError message to a separate test like `test_fromkeys_missing_iterable`, mark it as an expected failure, and file a new issue. After that, change the implementation of `fromkeys()` back to having `iterable` as required arg, so that it's a correct implementation apart from the message for the missing `iterable` arg.

This ensures that the contributor who fixes that issue later will easily discover that it fixes this case (due to test case unexpected success), and we definitely won't have the workaround left here in `fromkeys()` even when it's not needed.